mackerel/data_source_mackerel_monitor.go

```diff
@@ -197,6 +197,10 @@ func dataSourceMackerelMonitor() *schema.Resource {
 							Sensitive: true,
 							Elem:      &schema.Schema{Type: schema.TypeString},
 						},
+						"follow_redirect": {
+							Type:     schema.TypeBool,
+							Computed: true,
+						},
```

Comment:
[must] You need to add "follow_redirect" to the schema of the resource as well as the data source.

https://github.com/mackerelio-labs/terraform-provider-mackerel/blob/main/mackerel/resource_mackerel_monitor.go#L241

Reply:
Sorrry, I fix it 0598874fac535ad842aa346be6b13ba48c2cc783

Reply:
Thanks!